docs/user-guide/release-notes/master.rst

```diff
@@ -55,6 +55,27 @@ Known Issues
 Upgrade Instructions for 2.6.x --> master
 -----------------------------------------
 
+Upgrade the packages using:
+
+::
+
+    sudo yum update
+
+After yum completes you should migrate the database using:
+
+::
+
+    sudo -u apache pulp-manage-db
+
+After migrating the database reload the systemd process using:
+
+::
+
+    sudo systemctl daemon-reload
```

Comment:
I think this will fail on EL6 so it needs to say something like "if you are running systemd ....". Even if it doesn't fail it's not necessary on EL6. Maybe add it as a ..note block or something since it's not a required step for all platforms.